app/controlplane/pkg/data/membership.go

```diff
@@ -248,7 +250,7 @@ func (r *MembershipRepo) ListAllByUser(ctx context.Context, userID uuid.UUID) ([
 	mm, err := r.data.DB.Membership.Query().Where(
 		membership.MembershipTypeEQ(authz.MembershipTypeUser),
 		membership.MemberID(userID),
-	).All(ctx)
+	).WithOrganization().All(ctx)
```

Comment:
This Edge is deprecated, we cannot use it anymore. Note that now we have a polymorphic table, organizations are now identified by the `resourceID` field when `resourceType` is `organization`.

Reply:
Changed